src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,29 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        Book book = new Book();
+        book.setPrice(BigDecimal.valueOf(207));
+        book.setTitle("Sherlock Holmes");
+        book.setId(4L);
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        System.out.println("Creating book: " + bookDao.create(book));
+        System.out.println("Finding book: " + bookDao.findById(4L));
+        System.out.println("Deleting book: " + bookDao.deleteById(3L));
```

Comment:
Deleting a book with ID 3L without any pre-check is risky. You should first verify if the book with ID 3L exists.